package.json

```diff
@@ -15,22 +15,24 @@
   },
   "main": "lib/kv-redis.js",
   "scripts": {
-    "test": "mocha test/integration/*.js",
+    "test": "mocha test/unit/*.js && mocha test/integration/*.js",
```

Comment:
Well, when running the tests locally, I prefer to run fast unit-tests first before the slower integration tests.

Is there any reasonable way how to use different test command for dev vs. CI while supporting Windows too?